### PR TITLE
Add a example in README to show how to mock with fakefs

### PR DESCRIPTION
**Pull request by suzaku**

### CODE REVIEW

**Comment by jamescooke** [comment]
:+1: for this... I'm just about to pick over the commit.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,28 @@ work with pyfakefs.
...
+
+# Create a faked file system
+fs = fake_fs.FakeFilesystem()
+
+# Do some setup on the faked file system
+fs.CreateFile('/var/data/xx1.txt')
+fs.CreateFile('/var/data/xx2.txt')
+
+# Replace some built-in file system related modules you use with faked ones
+from mock import patch # Asumming you are using the mock library to ... mock things
```
Comment (jamescooke) [inline README.md]: `Asumming` > `Assuming`

**Comment by jamescooke** [comment]
Since `from mock import patch` is now `from unittest.mock import patch` in Python 3, it would be good to add this, but not required.

Fully approve - I would merge if I could.

**Comment by suzaku** [comment]
@jamescooke Thanks for reviewing, I've updated my commit.

**Comment by suzaku** [comment]
BTW, is this the "official" repo?

**Comment by jamescooke** [comment]
There isn't an official repo yet. See https://github.com/jmcgeheeiv/pyfakefs/issues/44

**Pull request merged**